frontend/public/lib/cookieOperations.js

```diff
@@ -40,3 +40,22 @@ export function getCookieProps(expiry) {
   console.log(cookieProps);
   return cookieProps;
 }
+
+export function getDisplaySignUpPromptFromCookie(cookieContent) {
+  const cookies = new Cookies();
+  // wasn't really sure how to handle the first time a user comes the page as the cookie being looked up is undefined at first
```

Comment:
That's fair! We could probably remove this comment, or keep its context in the PR description as opposed to the code.
```suggestion
```